matrixprofile/algorithms/snippets.py

```diff
@@ -45,6 +45,7 @@ def snippets(ts, snippet_size, num_snippets=2, window_size=None):
 
     """
     ts = core.to_np_array(ts).astype('d')
+    time_series_len = len(ts)
```

Comment:
You can just use "n" instead of creating a new variable.

Reply:
I think the value of n has been changed in line 81.

Reply:
You are right.